File: lib/nei/mobile/template/java/enum.java
//
//  {{type.name}}.java
//
//  Created by {{conf.author}}
//
//  Auto build by NEI Builder

package {{conf.aPkg}}.{{conf.mPkg}};

/**
 *  {{type.description}}
 */
public interface {{type.name}} {
    {% for attr in type.attrs %}
    {%- if attr.description -%}
    /**
     *  {{attr.description}}
     */
    {%- endif -%}
    public static final String {{attr.vkey}} = "{{attr.name}}";
    {% endfor %}
}

File: lib/nei/mobile/template/oc/enum.m
//
//  Enum.m
//
//  Created by {{conf.author}}
//
//  Auto build by NEI Builder

#import "{{conf.prefix}}ModelEnums.h"

{% for enu in enums -%}
/**
 * {{enu.description}}
 */
{% for attr in enu.pairs -%}
// {{attr.desc}}
NSString * const {{conf.prefix}}{{enu.name}}{{attr.key}} = @"{{attr.value}}";
{%- endfor %}
{% endfor %}